### ISSUE TITLE
config about ack mode

### ISSUE THREAD

**Issue by tabVersion** [ditsuke/google-pubsub-demo#1]
https://github.com/ditsuke/googlepubsub-demo/blob/8f8fe7af0fe7e308d352c8af4ee5c822b17b3de1/src/bin/consume.rs#L82-L85

shows you do ack on every message, which is unacceptable in production. 

Can we just ack the last message in the batch to replace the step?

**Comment by ditsuke** [comment]
On first look we should be able to batch ack multiple messages with the API `ReceivedMessage.ack()` is using itself internally https://github.com/yoshidan/google-cloud-rust/blob/main/pubsub/src/subscriber.rs#L37. Not sure yet if its publicly exposed.

**Comment by ditsuke** [comment]
Opened sibling issue upstream: https://github.com/yoshidan/google-cloud-rust/issues/49

**Comment by tabVersion** [comment]
Or can we just ignore the ack, just continuously pull data from the cloud without committing?

**Comment by ditsuke** [comment]
> Or can we just ignore the ack, just continuously pull data from the cloud without committing? 

No, I believe not. Pubsub will retain and send through those messages _again_ after an ack timeout unless acknowledged.
